packages/core/src/page.js

```diff
@@ -68,9 +68,14 @@ export class Page {
   }
 
   // Go to a URL and wait for navigation to occur
-  async goto(url, { waitUntil = 'load', cookies } = {}) {
+  async goto(url, { waitUntil = 'load', cookies, forceReload } = {}) {
     this.log.debug(`Navigate to: ${url}`, this.meta);
 
+    if (forceReload) {
+      this.log.debug('Navigating to blank page', this.meta);
+      await this.session.send('Page.navigate', { url: 'about:blank' });
```

Comment:
Do we not need to wait for navigation here ? does it work regardless ?

Reply:
about:blank is an empty page and does not load any external resources so i don't think we need to wait for it to load. it is instantaneous

Reply:
have we added tests for this ? or ^ is from chromium documentation ?